src/Backend/Modules/Slideshows/Actions/Add.php

```diff
@@ -80,4 +85,27 @@ private function handleForm()
             }
         }
     }
+
+    /**
+     * @return array
+     */
+    private function getPossibleTemplates()
+    {
+        $templates = array();
+        $finder = new Finder();
+        $finder->name('*.html.twig');
+        $finder->in(FRONTEND_MODULES_PATH . '/Slideshows/Layout/Widgets');
+        // if there is a custom theme we should include the templates there also
+        if (Theme::getTheme() != 'core') {
```

Comment:
@bjorvack  no templates will be found because you check if the dir exists 2 lines after this one. If the theme is 'core', the path will be `src/Frontend/Themes/core/Modules/Slideshows/Layout/Widgets` and this path does not exists, therefor finder won't go looking for templates in this path